types/passport-oauth2/index.d.ts

```diff
@@ -52,7 +52,7 @@ declare namespace OAuth2Strategy {
         verify(req: Request, state: string, meta: Metadata, callback: StateStoreVerifyCallback): void;
     }
 
-    type VerifyCallback = (err?: Error | null, user?: object, info?: object) => void;
+    type VerifyCallback = (err?: Error | null, user?: Express.User, info?: object) => void;
```

Comment:
I don't fully agree with this change that does not reflect the truth.
what you verify is the response from the oauth call and it's not yet an Express.User. What is an express.user is what is returned at the end of the VerifyCallback and can be dramatically different.
It's breaking for unnecessary reason I think